benchmark/realnest/03_create_table_combined_all_json_fields.benchmark

```diff
@@ -0,0 +1,33 @@
+# name: benchmark/realnest/03_create_table_combined_all_json_fields.benchmark
+# description: Create a table combining all the fields from a JSON file, unnesting all structures
+# group: [realnest]
+
+name create_table_combined_all_json_fields
+group real_nest
+
+require json
+
+require httpfs
+
+cache real_nest.duckdb
+
+load benchmark/realnest/load.sql
+
+run
+CREATE OR REPLACE TABLE combined AS (SELECT unnested_hlt.*,
+    unnested_pv.*,
+    unnested_met.*,
+    unnested_muon.*,
+    unnested_electron.*,
+    unnested_tau.*,
+    unnested_photon.*,
+    unnested_jet.*
+    FROM unnested_hlt
+    LEFT JOIN unnested_pv ON unnested_hlt.rowid = unnested_pv.rowid
+    LEFT JOIN unnested_met ON unnested_hlt.rowid = unnested_met.rowid
```

Comment:
So for this benchmark, the `unnested_*` tables have already been unnested in the load step. 
I think it might be better if they are unnested in the query to avoid confusion. 
So maybe the query could look like
```
Create or replace table combined as 
(select 
    unnested_pv.*
    unnested_met.*
    ....
    FROM 
        (SELECT rowid, UNNEST(htl) as hlt FROM run2012B_singleMu) as unnested_hlt 
        LEFT JOIN
        (SELECT rowid, UNNEST(pv) AS pv FROM run2012B_singleMu) as unnested_pv ON  unnested_hlt.row_id = unnested_pv.row_id 
        LEFT JOIN 
        (SELECT rowid, UNNEST(met) AS met FROM run2012B_singleMu) as unnested_met ON  unnested_hlt.row_id = unnested_met.row_id 
        ....
```

That way it is clear to future readers where the unnesting is happening

Reply:
You can then remove all of the `create table` statements in the `load.sql` that are basically doing the same thing